apps/standalone/deploy/helm/flightctl-ui/templates/flightctl-ui-certs-secret.yaml

```
@@ -2,7 +2,8 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: flightctl-ui-certs
-  namespace: {{ .Values.flightctlUi.namespace }}
+  namespace: {{ .Release.Namespace }}
 type: Opaque
 stringData:
-  ca.crt: {{ .Values.flightctlUi.certs.ca | quote }}
\ No newline at end of file
+  ca.crt: {{ .Values.api.caCert | quote }}
+  ca_oidc.crt: {{ default (.Values.global).flightctl.auth.caCert .Values.auth.caCert | quote }}
```

```suggestion
  ca_oidc.crt: {{ default (.Values.global).flightctl.auth.caCert .Values.auth.caCert | quote }}
```
Just curious, why the braces? are those mandatory?

---

if `global` object is not defined we wont fail here. Without braces, the deployment fails with nil pointer exception

---

nice I didn't know about that.